scheduler/src/managers/scheduler/schedule.rs

```diff
@@ -0,0 +1,40 @@
+use crate::managers::scheduler::errors::SchedulerError;
+use orka_proto::node_agent::{Workload, WorkloadStatus as AgentWorkloadStatus};
+use orka_proto::scheduler_controller::workload_status::Resources;
+use orka_proto::scheduler_controller::{
+    SchedulingRequest, WorkloadStatus as SchedulerWorkloadStatus,
```

Comment:
The workload status is sent to, and handled by, the Controller. I would advise renaming it here, and in other places in this file.

```suggestion
    SchedulingRequest, WorkloadStatus as ControllerWorkloadStatus,
```